src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/MemberRepository.cs

```diff
@@ -61,24 +76,53 @@ public async Task<QueryResult<Member>> GetPotentialChannelMembersAsync(Guid chan
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var whereCondition = @"WHERE Id NOT IN (
-                                        SELECT ChannelMembers.MemberId
-                                        FROM Members
-                                        INNER JOIN ChannelMembers
-                                        ON Members.Id = ChannelMembers.MemberId
-                                        WHERE ChannelMembers.ChannelId = @channelId)" +
-                                        (!string.IsNullOrEmpty(nameFilter) ? " AND LOWER(Members.Name) LIKE LOWER('%' + @nameFilter + '%')" : "");
-
-                var sqlQuery = @"SELECT *  
-                                 FROM Members " +
-                                 whereCondition +
-                               @" ORDER BY Name
-                                 OFFSET @pageSize * (@pageNumber - 1) ROWS
-                                 FETCH NEXT @pageSize ROWS ONLY
-
-                                 SELECT COUNT(*) 
-                                 FROM Members " +
-                                 whereCondition;
+                var sqlQuery = $@"
+                    SELECT 
+                        {nameof(Member.Id)},
+                        {nameof(Member.Email)},
+                        {nameof(Member.IsAfk)},
+                        {nameof(Member.IsBanned)},
+                        {nameof(Member.LastActivity)},
+                        {nameof(Member.LastNudged)},
+                        {nameof(Member.Name)},
+                        {nameof(Member.PhotoName)},
+                        {nameof(Member.Role)},
+                        {nameof(Member.SaasUserId)},
+                        {nameof(Member.Status)},
+                        {nameof(Member.IsActive)}
+                    FROM 
+                        Members 
+                    WHERE 
+                        {nameof(Member.Id)} NOT IN (
+                            SELECT 
+                                ChannelMembers.{nameof(ChannelMember.MemberId)}
+                            FROM 
+                                Members
+                            INNER JOIN ChannelMembers
+                                ON Members.{nameof(Member.Id)} = ChannelMembers.{nameof(ChannelMember.MemberId)}
+                            WHERE 
+                                ChannelMembers.{nameof(ChannelMember.ChannelId)} = @{nameof(channelId)})
+                                {(!string.IsNullOrEmpty(nameFilter) ? $" AND LOWER(Members.{nameof(Member.Name)}) LIKE LOWER('%' + @{nameof(nameFilter)} + '%')" : "")}
```

Comment:
Let's replace nameFilter on nameof(nameFilter)

Reply:
Which place are you pointing?